machine.go

```diff
@@ -441,11 +442,67 @@ func (m *Machine) startup(command string, extracontent [][2]string) (int, error)
 	if mergedUsrSystem() {
 		prefix = "/usr"
 	}
-	w.CopyFile(prefix + "/lib/x86_64-linux-gnu/libc.so.6")
-	w.CopyFile(prefix + "/bin/busybox")
 
-	/* Amd64 dynamic linker */
-	w.CopyFile("/lib64/ld-linux-x86-64.so.2")
+	qemu := ""
+
+	// Useful references:
+	// https://salsa.debian.org/go-team/compiler/golang/blob/golang-1.10/debian/helpers/goenv.sh
+	// https://sourceware.org/glibc/wiki/ABIList
+	switch runtime.GOARCH {
```

Comment:
Please rework this to use table-like data structure instead of switch-case approach.
May be it is useful to define the single structure related to qemu here and use it in go-debos/debos/pull/91 ?